app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'authorId'
+  belongs_to :post, foreign_key: 'postId'
+
+  def update_likes_counter
+    post = Post.find(postId)
+    post.update(likesCounter: Like.where(postId: post.id).count)
+  end
```

Comment:
- [ ] [OPTIONAL] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new `like` has been saved to the database, then you can make the `#update_likes_counter` method private.